src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public interface Storage {
+    public void add(String fruit, int quantity);
+
+    public void change(String fruit, int newQuantity);
```

Comment:
Similarly, remove the 'public' modifier here as well for consistency and to adhere to the encapsulation principle.